src/System.Private.CoreLib/shared/System/Runtime/InteropServices/ComSourceInterfacesAttribute.cs

```diff
@@ -15,7 +15,7 @@ public ComSourceInterfacesAttribute(string sourceInterfaces)
 
         public ComSourceInterfacesAttribute(Type sourceInterface)
         {
-            Value = sourceInterface.FullName;
+            Value = sourceInterface.FullName!;
```

Comment:
FullName is nullable?

Reply:
As it is nullable in parent class i was need to put `?` for all descendants to shut compiler error, even if it was returning non null value, but as @stephentoub suggested i will update ones not returning null by suppressing warnings